Sources/Workspace/InitPackage.swift

```diff
@@ -15,7 +15,31 @@ import PackageModel
 public final class InitPackage {
     /// The tool version to be used for new packages.
     public static let newPackageToolsVersion = ToolsVersion(version: "5.2.0")
-    
+
+    /// Options for the template package.
+    public struct InitPackageOptions {
+        /// The type of package to create.
+        public var packageType: PackageType
+
+        /// Whether to laydown files related to corelibs-xctest manifest.
+        public var enableXCTestManifest: Bool
+
+        /// The list of platforms in the manifest.
+        ///
+        /// Note: This should only contain Apple platforms right now.
```

Comment:
Is that because those are the only ones that (at least AFAIK) are versioned right now?

Reply:
Yes, only Apple platforms have support for declaring a minimum deployment target version.